src/helpers/renderWithTheme.js

```diff
@@ -11,10 +11,12 @@ const emotionCache = createCache({
 
 emotionCache.compat = true
 
-export default (component, options) =>
+export default (component, options, modifyTheme = {}) =>
```

Comment:
Here you receive theme in option

Why not override the current theme ? Instead of extending it ?